### ISSUE TITLE
Minor modification to the paper

### ISSUE THREAD

**Issue by letiziagambacorta1** [NicOudart/PyBWE#3]
Line 11 and 16: I suggest to use the word ‘enhanced’ instead of ‘better,  which is  more generic.
Line 31 and 32: Here it is necessary to underline the distinction between the signal model (AutoRegressive/ ARMA) and the method for estimation of the coefficients (Burg algorithm). The way it is written now, it seem the Burg algorithm to be a way to model an AR model instead of a way to solve the Yule-Walker equation.  
Line 46: This is more a question on the method and does not have to be included in the paper. How is pursued the determination of an adequate number of singular values?
Line 66: It is worthy citing that BWI has been applied to SHARAD data for EMI reconstruction.

**Comment by letiziagambacorta1** [comment]
Extra References on Radar Sounders:
More mature paper on BWE applied to SHARAD data: Raguso, M. C., Mastrogiuseppe, M., Gambacorta, L., Di Achille, G., & Seu, R. (2023). Range resolution enhancement of SHAllow RADar (SHARAD) data via bandwidth extrapolation technique: Enabling new features detection and improving geophysical investigation. Icarus, 115803. 
Raguso, M. C., Piazzo, L., Mastrogiuseppe, M., Seu, R., & Orosei, R. (2018, September). Resolution enhancement and interference suppression for planetary radar sounders. In 2018 26th European Signal Processing Conference (EUSIPCO) (pp. 1212-1216). IEEE.
Piazzo, L., Raguso, M. C., Seu, R., & Mastrogiuseppe, M. (2019). Signal enhancement for planetary radar sounders. Electronics Letters, 55(3), 153-155.

**Comment by NicOudart** [comment]
Thank you for your review of the paper. I updated the paper.md and paper.bib files following your comments:

Line 11 and 16: I modified "better" with "enhanced".
Line 31 and 32: I agree that my description here can be misleading. I tried to clarify the distinction between the model and the method for coefficients estimation. To avoid repetitions, I also modified the 'PyBWE', 'PyPBWE' and 'PySSBWE' presentation.
Line 66: I now cite the EMI removal in SHARAD data.
Additional references: Thank you for the (Raguso et al., 2023) reference, it was not published yet when I began writing the article, and I forgot to add it later. I also now cite the (Piazzo et al., 2019) reference to introduce BWI as a way to remove EMI.

Are these modifications ok for you?

Concerning your technical question:

Line 46: The adequate number of singular values to separate the signal and noise sub-spaces is determined by Akaike's Information Criterion by default, as recommended by (Piou, 1999). This parameter can also be set manually by the user. We plan to investigate other criteria for future versions of PySSBWE.
